<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>1013</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Surface Pinhole Picker v.0.51, 23/11/2016</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>C., Sumix, 2016</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>150</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="btnPath1">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="surfacePath"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Select data file</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="btnLoadSurface">
        <property name="text">
         <string>Load Surface</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="gvSurfImage">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="7">
           <widget class="QPushButton" name="btnLastFrame">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="btnFirstFrame">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="btnFrameForvard">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="btnFrameBackward">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="10">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="btnStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Stop</string>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
	background-image: url(:/icons/img/ico-stop.png);
}
*:hover{
	background-image: url(:/icons/img/ico-stop-hover.png);
}
*:pressed{
	background-image: url(:/icons/img/ico-stop-down.png);
}
*:disabled{
	background-image: url(:/icons/img/ico-stop-disabled.png);
}
</string>
            </property>
            <property name="text">
             <string>S</string>
            </property>
            <property name="type" stdset="0">
             <string notr="true">icon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnPlay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>24</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Play/Pause</string>
            </property>
            <property name="styleSheet">
             <string notr="true">*{
	background-image: url(:/icons/img/ico-run.png);
}
*:hover{
	background-image: url(:/icons/img/ico-run-hover.png);
}
*:pressed{
	background-image: url(:/icons/img/ico-run-down.png);
}
*:disabled{
	background-image: url(:/icons/img/ico-run-disabled.png);
}


*:checked{
	background-image: url(:/icons/img/ico-pause.png);
}
*:checked:hover{
	background-image: url(:/icons/img/ico-pause-hover.png);
}
*:checked:pressed{
	background-image: url(:/icons/img/ico-pause-down.png);
}
*:checked:disabled{
	background-image: url(:/icons/img/ico-pause-disabled.png);
}</string>
            </property>
            <property name="text">
             <string>P</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="type" stdset="0">
             <string notr="true">icon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="13">
           <widget class="QLabel" name="lblTotalFrames">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <widget class="QComboBox" name="cbPlaySpeed">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>5</number>
            </property>
            <property name="maxVisibleItems">
             <number>11</number>
            </property>
            <item>
             <property name="text">
              <string>0.1x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.2x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.4x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.6x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>0.8x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8x</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10x</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="11">
           <widget class="QLabel" name="lblCurrentFrame">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="8">
           <widget class="QSlider" name="sliderFrame">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="12">
           <widget class="QLabel" name="lblSeparator2">
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="rightMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="gvPinHole">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblPixCoord">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>(x-y)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10"/>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Y coord</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnShowPinhole">
              <property name="text">
               <string>Show Pinhole</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="sbY">
              <property name="maximum">
               <number>1600</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X coord</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sbX">
              <property name="maximum">
               <number>2500</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGraphicsView" name="gvHisto"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
